.github/workflows/build-cirros-bionic.yaml

```diff
@@ -0,0 +1,189 @@
+name: CirrOS image builder
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build-cirros-aarch64:
+    name: aarch64
+    runs-on: ubuntu-18.04
+    env:
+      ARCHES: aarch64
+      BOOTTEST: "true"
+      QUIET: 1
+    steps:
+      - name: Pull cirros source artifacts
+        uses: actions/checkout@v3
+
+      - name: Build cirros-aarch64
+        shell: bash
+        run: |
+          sudo apt-get install -y cloud-utils qemu-system openbios-ppc
+          ./bin/system-setup
+          ./bin/build-release daily
+          rm -rf ../build-*/build ../build-*/stage ../build-*/tmp
+          tar -cJf cirros-aarch64.tar.xz ../build-*
+
+      - name: Upload cirros-aarch64 build artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: cirros-aarch64
+          path: cirros-aarch64.tar.xz
```

Comment:
Just list release/ contents here. People fetch single files not whole build directory.

Reply:
So the original reason I uploaded the tarball instead of just the release directory was because of limitations stated by the `upload-artifact` action: https://github.com/actions/upload-artifact#limitations.

I was not sure if there were any particular permissions we wanted to preserve on the images, and I also wanted to prevent any case-insensitive related issues later down the line.